dask/bytes/core.py

```diff
@@ -50,6 +50,11 @@ def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize=2**27,
     sample : bool or int
         Whether or not to return a header sample. If an integer is given it is
         used as sample size, otherwise the default sample size is 10kB.
+    include_path : bool
+        Whether or not to include the path with the bytes representing a particular file.
+        If True ``blocks`` is a list of tuples where the first item is path and the second
+        is a list of ``dask.Delayed`` where each delayed object computes to a
+        block of bytes from that file.
```

Comment:
We might also consider leaving blocks as-is, but adding a new list of paths that has the same length.

I suspect that @martindurant has preferences here.

Reply:
Yeah I couldn't figure out which approach was better so I decided to stop waffling and do something. Definitely open to suggestions and preferences :)